inst/plumber/v1/endpoints.R

```diff
@@ -9,23 +9,25 @@ library(pipapi)
 #* Ensure that version parameter is correct
 #* @filter validate_version
 function(req, res) {
-  # tictoc::tic("filters")
-  # browser()
-  if (!is.null(req$argsQuery$version) & !grepl("swagger", req$PATH_INFO)) {
-    if (!is.null(req$argsQuery$version)) {
-      if (!req$argsQuery$version %in% lkups$versions) {
+
+  #If no arguments are passed, use the latest version
+  if (is.null(req$argsQuery$release_version) && is.null(req$argsQuery$ppp_version) &&
+     is.null(req$argsQuery$version) && is.null(req$argsQuery$identity)) {
+      version <- lkups$latest_release
```

Comment:
I have finally replaced the `&&` by `&`. We are still using 4.0.2 in PROD (soon to be updated to 4.1.2) and `&` is significantly faster than `&&` (even if we are in nanoseconds territory)